packages/app-js/src/snippets/transfer.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017-2019 @polkadot/app-123code authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default `// transfer
```

Comment:
Yes, I need to turn this into a proper snippet - will do that next round along with the queries one I added earlier.